src/Components/LoginLinkWrapper.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import styled from 'styled-components/macro';
+import { flexCenter } from '../style/mixin';
+import { Link } from 'react-router-dom';
+
+export default function LoginLinkWrapper(props) {
+  const [isImageHover, setIsImageHover] = useState(false);
+  return (
+    <>
+      <LoginLink
+        to={props.url}
+        onMouseOver={() => setIsImageHover(true)}
+        onMouseOut={() => setIsImageHover(false)}
+      >
+        <LoginIcon
+          src={
+            isImageHover
+              ? `http://localhost:3000/images/${props.icon}`
+              : `http://localhost:3000/images/${props.iconGray}`
+          }
+        />
```

Comment:
```suggestion
        <LoginIcon
          src={
            isImageHover
              ? `images/${props.icon}`
              : `images/${props.iconGray}`
          }
        />
```
localhost 부분은 삭제해주셔도 됩니다!